tests/EventExport/Format/HTML/HTMLFileWriterTest.php

```diff
@@ -238,7 +238,7 @@ public function it_shows_taaliconen()
                 'price' => 'Niet ingevoerd',
                 'dates' => 'ma 02/03/15 van 13:30 tot 16:30  ma 09/03/15 van 13:30 tot 16:30  '.'ma 16/03/15 van 13:30 tot 16:30  ma 23/03/15 van 13:30 tot 16:30  '.'ma 30/03/15 van 13:30 tot 16:30 ',
                 'taalicoonCount' => 1,
-                'taalicoonDescription' => 'Je spreekt nog geen of niet zo veel Nederlands.',
+                'taalicoonDescription' => 'Je begrijpt of spreekt nog niet veel Nederlands.',
```

Comment:
Nit: We could use the `TaalicoonDescription` constants here and in the other tests. But since it's just tests it's also fine to leave it as-is.